Form/Type/SeoMetadataType.php

```diff
@@ -69,8 +76,15 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $class = null;
+        if ($this->container->has('cmf_seo.backend_type_phpcr')) {
```

Comment:
@dbu you see the issue with the parameter `cmf_seo.backend_type_phpcr` now? This one ist not available, when `ORM` is used `cmf_seo.backend_type_orm` instead. Both are just set to true, and have no false as default.